recipes/emacs/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+if [ "$(uname)" == "Darwin" ]; then
+  OPTS="--without-x"
+else
+  OPTS="--x-includes=$PREFIX/include --x-libraries=$PREFIX/lib"
+fi
+
+bash configure  --prefix=$PREFIX $OPTS
```

Comment:
Could we please try `chmod +x configure` and `./configure`? If that doesn't work, I would be extremely surprised, but maybe it doesn't.